src/test/kotlin/racing/junit/CarTest.kt

```diff
@@ -0,0 +1,31 @@
+package racing.junit
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+import racing.condition.FirstRoundCondition
+import racing.model.Car
+
+class CarTest {
+    @ParameterizedTest
+    @ValueSource(ints = [1])
+    fun `doRacing 성공`(round: Int) {
+        val conditions = listOf(FirstRoundCondition())
+        val car = Car(0)
+
+        car.doRacing(round, conditions)
+
+        Assertions.assertThat(car.dist).isEqualTo(1)
+    }
```

Comment:
Assertions는 static import를 하면 제거할 수 있어요 🙂
```suggestion
        assertThat(car.dist).isEqualTo(1)
    }
```